src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -38,10 +37,35 @@
 .featureSymbol {
   font-weight: $bold;
 }
-.stableId{
+.stableId {
   font-weight: $bold;
 }
-.featureSymbol + .stableId{
+.featureSymbol + .stableId {
   margin-left: 12px;
   font-weight: $light;
 }
+
+.downloadRow {
+  > .value {
+    display: flex;
+    position: relative;
+    width: 400px;
+  }
+}
+
+.downloadLink {
+  color: $blue;
+  cursor: pointer;
+}
+
+.downloadWrapper {
+  display: inline-block;
```

Comment:
The inline-block rule doesn't seem to change anything. `display:flex` of the parent is already positioning the two elements next to one another.

Reply:
✅